vscode-trace-extension/src/trace-explorer/available-views/trace-explorer-available-views-webview-provider.ts

```diff
@@ -47,6 +49,12 @@ export class TraceExplorerAvailableViewsProvider implements vscode.WebviewViewPr
 	    // Handle messages from the webview
 	    webviewView.webview.onDidReceiveMessage(message => {
 	        switch (message.command) {
+	        case 'connectionStatus':
+	            if (message.data && message.data.status) {
+	                const status: boolean = JSON.parse(message.data.status);
```

Comment:
You don't need JSON.parse(), after removing JSON.stringify() at sending time.

Reply:
Not necessarily, according to my local testing of both approaches. Here, not explicitly parsing led to the wrong status being rendered (misinterpreted in here). It ends up looking like we do have to explicitly stringify-then-parse the boolean value, across; so stated in the commit message and PR description.

Reply:
After removing the JSON conversion update as followed:
```
if (message.data && message.data.serverStatus !== undefined) {
    this._statusService.render(message.data.serverStatus);
}
```

Reply:
Let's keep it as-is. It works fine.